src/main/kotlin/ai/digital/integration/server/common/domain/Cluster.kt

```diff
@@ -8,6 +8,7 @@ import org.gradle.kotlin.dsl.property
 open class Cluster(objects: ObjectFactory) {
```

Comment:
There is no documentation about it

Reply:
And please create a blog post for that to explain on the high level how it works.